src/components/TeacherComponents/TermsAgreementForm/TermsAgreementForm.tsx

```diff
@@ -47,6 +48,7 @@ const TermsAgreementForm: FC<TermsAgreementFormProps> = ({
   showSavedDetails = false,
   handleEditDetailsCompletedClick = () => {},
   showPostAlbCopyright = true,
+  updatedAt,
```

Comment:
sorry to nitpick but can this be copyrightYear as well, bit confusing otherwise!

Reply:
No probs, have just pushed up the change